index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf6">PF6.</a> A way to associate datatypes with values such as
+dates and times.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf7">PF7.</a> A facility to express one or more directed graphs,
+such as a social network, in a single document.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf8">PF8.</a> Supports signature sets.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf9">PF9.</a> Embeddable in HTML such that search crawlers will
+index the machine-readable content.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf10">PF10.</a> Data on the wire is easy to debug and serialize to
+database systems.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf11">PF11.</a> Nesting signed data does not cause data size to
+double for every embedding.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf12">PF12.</a> Proof format supports Zero-Knowledge Proofs.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf13">PF13.</a> Proof format supports arbitrary proofs such as Proof
+of Work, Timestamp Proofs, and Proof of Stake.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf14">PF14.</a> Proofs can be expressed unmodified in other data
+syntaxes such as YAML, N-Quads, and CBOR.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf15">PF15.</a> Changing key-value ordering, or introducing
+whitespace does not invalidate signature.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf16">PF16.</a> Support experimental signature systems.
+          </td>
+          <td class="missing">✖</td>
```

Comment:
This is factually incorrect. You can support any experimental signature system. Many people including uPort and Sovryn's implementations use experimental signature systems.

Reply:
The explanation elaborates on what this one liner is getting at:

> The LD-Proof format is naturally extensible, not requiring the format to be
extended in a formal international standards working group in order to
prevent namespace collisions. The JWT format requires entries in a centralized
registry and does not support experimentation as easily as the LD-Proof format
does. LD-Proof format extension is done through the decentralized publication
of cryptographic suites that are guaranteed to not conflict with other LD-Proof
extensions. This approach enables developers to easily experiment with new
cryptographic signature mechanisms that support selective disclosure,
zero-knowledge proofs, and post-quantum algorithms.

Is there concrete specification text that you can suggest that would summarize that paragraph?

Reply:
I have tried to make the statement more aligned with the paragraph text in 12c109d39c56eccc9b5a6e8a79df85228a094451.